account/lnurl.go

```diff
@@ -0,0 +1,62 @@
+package account
+
+import (
+	"encoding/json"
+	"errors"
+	"math"
+	"net/http"
+
+	"github.com/breez/breez/data"
+	lnurllib "github.com/fiatjaf/go-lnurl"
```

Comment:
Can you rename the package golnurl or lnrlgo and use it in the app without aliasing?

Reply:
That would be wrong as this package is being used in many other places and they would all break. But I can refactor so it's used as `lnurl` and doesn't conflict with other variables. Is that good?

Reply:
Yes. That would be better. Thanks a lot!